vendor/assets/stylesheets/rolodex/settings/utilities/lib/_position.sass

```diff
@@ -0,0 +1,48 @@
+// Position
+// ========================================
+
+.p-relative
+  position: relative
+
+.p-absolute
+  position: absolute
+
+.p-fixed
+  position: fixed
+
+// Z-indexes
+
+.z-100
+  z-index: 100
+
+.z-500
+  z-index: 500
+
+.z-1000
+  z-index: 1000
+
+// Fill
+
+.fill
+  top: 0
+  right: 0
+  bottom: 0
+  left: 0
+
+.fit
+  max-width: 100%
+
+// Absolute centering
```

Comment:
I think this will be duplicate of https://github.com/bellycard/rolodex/blob/master/vendor/assets/stylesheets/rolodex/settings/mixins/_layout.sass#L37.